lib/manager/helm-values/util.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Determine whether a given partial Helm values.yaml object potentially defines
+ * a Docker dependency.
+ *
+ * There is no exact standard of how Docker dependencies are defined in Helm
+ * values.yaml files (as of January 1st 2020), this function defines a
+ * heuristic based on the most commonly used format in the stable Helm charts:
+ *
+ * image:
+ *   repository: 'something'
+ *   tag: v1.0.0
+ */
+export function matchesHelmValuesDockerHeuristic(
+  parentKey: string,
+  data: any
+): boolean {
```

Comment:
```suggestion
): is DockerImage{
```

```ts
type DockerImage = {
  repository: string,
  tag: string
};
```
Then you can use this function as type guard

Reply:
Thanks, that worked nicely.

I renamed the suggested type to `HelmDockerImageDependency` as I felt it would be nice to communicate that this format is only used with Helm (at least that's the only situation I know of).

I also had to add a small helper function to check the existence of keys in combination with `unknown`, not sure if there is a better way. See:

https://github.com/microsoft/TypeScript/issues/21732
https://stackoverflow.com/a/58630274

Would be great if you could give that a final look.